src/components/Navbar.jsx

```diff
@@ -36,11 +36,29 @@ const WalletPreview = ({ wallet, walletInfo, unit, showBalance }) => {
 
 const CenterNav = ({ makerRunning, onClick }) => {
   const { t } = useTranslation()
+  const serviceInfo = useServiceInfo()
 
   const scheduleEnabled = isFeatureEnabled('schedule')
+  const [collaborativeOperationRunning, setCollaborativeOperationRunning] = useState(false)
+
+  useEffect(() => {
+    const coinjoinInProgress = serviceInfo && serviceInfo.coinjoinInProgress
+    setCollaborativeOperationRunning(coinjoinInProgress)
+  }, [serviceInfo])
```

Comment:
Would it work to just pass in `coinjoinInProgress` as a prop to the `CenterNav` component similar to what we do with `makerRunning`?